R/two_class_diag_plots.R

```diff
@@ -20,18 +20,15 @@ plot_twoclass_obs_pred <-
     prob_name <-
       first_class_prob_name(dat, event_level, y_name)
     # plotting
-    p <- dat %>%
+    dat %>%
```

Comment:
Is the TODO still needed?

Reply:
Yes, I fixed this in the multiclass one, I will fix it here as well :)